### DISCUSSION TITLE
Is it possible in Tailwind to convert its classes once app is built to single unique classname ?

### DISCUSSION THREAD

**Discussion by asotog** [tailwindlabs/tailwindcss#14466] [Help]
Hi all,

Can you transform tailwind classes into unique CSS classes, like the names generated when using CSS modules  ? 

### Example
Instead of
```
<div class="flex" ...
```
like
```
<div class="flex__JGj5N" ...
```

### Problem

A third-party vendor injecting its code into our site to style a form is creating conflicts with Tailwind classes, for example, 3rdparty vendor implemented a .flex class which is affecting the Tailwind .flex class

I know it is possible to use prefix property on the tailwind config, but that will require updating the entire codebase with the new prefix for each single class

Thanks in advance,

**Comment by matagaralph** [comment]
https://github.com/tailwindlabs/tailwindcss/discussions/10593 Check this one they is a discussion that has been going around here i suppose it's the same topic.
